app/jinja_filters.py

```diff
@@ -145,14 +145,19 @@ def format_list(list_items):
     return summary_list
 
 
+@blueprint.app_template_filter()
+def format_list(list_items, delimiter=', '):
```

Comment:
Does the changing of the name of the function above imply that this list is ordered?
Is this not more like `join_list`?

Reply:
@jonnyshaw89 it more implies this is the main list formatting method, but we can rename it `format_join_list`?

Reply:
i don't see its more of a list than the other, just different ways of formatting a list.
If you look at it in the context for address, is it a list or is it a formatting style?

Reply:
Well it's listing the address 🙃